contracts/erc20-gateway/ERC20Cogateway.sol

```diff
@@ -213,6 +210,7 @@ contract ERC20Cogateway is
             "Withdrawal amount should be greater than max reward."
         );
 
+        address valueToken = UtilityTokenInterface(_utilityToken).valueToken();
```

Comment:
why is L195 not already a `UtilityTokenInterface`

Reply:
Yes, it can be changed. 
Changes are done.